File: basics/q7/solution.py
def grade(marks):
    # A -> 90 and above , B -> 80 above , C -> 70 above , D -> 60 above 

    if marks >= 90:
        return "Grade A"
    elif marks >= 80:
        return "Grade B"
    elif marks >= 70:
        return "Grade C"
    elif marks >= 60:
        return "Grade D"
    else:
        return "Fail"

x = int(input("Enter the marks: "))
print(grade(x))

File: basics/q7/solution.cpp
#include<iostream>
using namespace std;

// A -> 90 and above , B -> 80 above , C -> 70 above , D -> 60 above 

string grade(int marks){
    if(marks >= 90){
        return "Grade A";
    }
    if(marks >= 80){
        return "Grade B";
    }
    if(marks >= 70){
        return "Grade C";
    }
    if(marks >= 60){
        return "Grade D";
    }
    else{
        return "Fail";
    }
}

int main(){
    int y;
    cout << "Enter the marks: " << endl;
    cin >> y;
    cout << grade(y) << endl;

    return 0;
}